jwql/website/apps/jwql/views.py

```diff
@@ -258,8 +258,8 @@ def api_landing(request):
     return render(request, template, context)
 
 
-@auth_required
-def archived_proposals(request, user, inst):
+# @auth_required
+def archived_proposals(request, inst): #request, user, inst):
```

Comment:
Nice use of the workaround to get through the login firewall.  We should just make sure to revert this back before merging this PR.

Reply:
Reverted